llvm/test/MC/AArch64/ilp32-diagnostics.s

```diff
@@ -85,6 +85,14 @@
 
    ldr x24, [x23, #:got_lo12:sym]
 // CHECK-ERROR: error: ILP32 64-bit load/store relocation not supported (LP64 eqv: LD64_GOT_LO12_NC)
+// CHECK-ERROR: ^
```

Comment:
Testing caret `^` is not really useful... To test diagnostics, just check the file and line information before `error:`

I'll adjust other diagnostics in the file.

Reply:
Yes, I just followed the same pattern as other test cases in the file. I agree that this is not useful w/o `--strict-whitespace --match-full-lines`. I'll delete this, thanks.

Reply:
Fixed in merge commit 83cd3ca82ba0f31bbedf16112236cffb26a3a73b